camayoc/ui/models/pages/scans.py

```diff
@@ -15,10 +15,10 @@
 
 
 class ScanListElem(AbstractListItem):
-    def download_scan(self) -> Download:
+    def download_scan(self, timeout: int) -> Download:
         scan_locator = "td[class*=-c-table__action] button[data-ouia-component-id=download]"
         timeout_start = time.monotonic()
-        while 10 * 60 > (time.monotonic() - timeout_start):
+        while timeout > (time.monotonic() - timeout_start):
```

Comment:
I wonder if it's worth it to create a fixture and then pass the value through multiple layers, when `settings` object is globally available and we already liberally use it in tests, framework and helpers?

In fact, [UI client already holds a reference to settings](https://github.com/quipucords/camayoc/blob/main/camayoc/ui/client.py#L71), and all pages have a private reference to client. So you can just do:

```suggestion
        timeout = self._client._camayoc_config.camayoc.scan_timeout
        while timeout > (time.monotonic() - timeout_start):
```

Reply:
OK, let's make it simpler.